apps/api/src/middleware/openapi-meta.ts

```diff
@@ -38,6 +38,11 @@ export const openapiMeta: OpenAPIObjectConfigure<{ Bindings: Env }, string> = {
       description:
         "Course data, such as department, school, instructors, and previous sections. Sourced from the UCI Course Catalog and WebSoc.",
     },
+    {
+      name: "Programs",
+      description:
+        "Program data regarding currently available majors, minors, and specializations, as well as their respective requirements. Sourced from the UCI General Catalog and WebSoc.",
```

Comment:
This isn't actually sourced from either of those places, and sadly we have to be a bit opaque about where they do come from. Something like "Data on majors, minors, and specializations and their requirements" is probably all we can say.